src/fontra/backends/designspace.py

```diff
@@ -1072,6 +1076,40 @@ def _writeDesignSpaceDocument(self):
         self.dsDoc.write(self.dsDoc.path)
         self.dsDocModTime = os.stat(self.dsDoc.path).st_mtime
 
+    async def getBinaryData(self) -> dict[str, bytes]:
```

Comment:
So this would return *all* binary data for *all* layers. That's insanely inefficient: we *always* load "big things" on demand. But see the general comment, you ignored the method signature that I prescribed.